README.md

```diff
@@ -323,6 +324,47 @@ docker rm sbot
 ./create-sbot
 ```
 
+## kubernetes setup
+
+Yaml config files for Kubernetes are included in this repository at
+`hack/k8s/deployment.yaml`. This has been tested on DigitalOcean. Tweaks may be
+necessary for clusters hosted by other providers.
+
+The SSB config file is injected into the container as a volume mount. You should
+update the `ssb-config` configMap in the deployment file with your external IP
+or domain, and any other changes you require. Unfortunately, because this is
+being done on a subPath, changes to the configMap will not propogate
+automatically. Please keep this in mind if you find you need to make config
+changes, or, pull requests welcome.
+
+After updating the config, you can install SSB into your cluster with the
+following:
+
+```shell
+kubectl apply -f hack/k8s/deployment.yaml
+```
+
+This will do a number of things:
+
+1. create a namespace on your cluster called `scuttlebutt`
+1. create a service exposing port 8008
+1. create a 5gb persistent volume which is mapped to `/home/node/.ssb`
```

Comment:
what happens if you go over 5gb? i ask because all my pubs use more than 5gb.

Reply:
Super great question. You're way more active than I am, I'm less than 4 after 2 years. Maybe start bigger? Worth thinking about.

Reply:
I've added some notes to the readme calling this out.